### ISSUE TITLE
Support glmmTMB?

### ISSUE THREAD

**Issue by bwiernik** [goodekat/ggResidpanel#10]
The glmmTMB package (https://github.com/glmmTMB/glmmTMB) is a relatively recent addition to R's mixed effects models landscape that combines much of the functionality of both nlme and lme4. I was wondering if support for glmmTMB models could be added?

glmmTMB models have basic structure similar to lmerMod models, so it looks like supporting them wouldn't entail much more than adding "glmmTMB" to the list of supported models in the check functions.

**Comment by goodekat** [comment]
Good idea! I'm pretty busy right now (trying to finish up my PhD), but since this does seem like it would be a relatively easy additional, I'll try to take a look at it in the near future.
